src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -125,13 +125,12 @@ const RenderListFeed = (props) => {
 
   const isBlurred = item?.isBlurredPost && item?.anonimity;
 
-  const commentHeight = () => {
+  const addCommentPreviewHeight = () => {
+    const commentSectionHeight = getHeightReaction() - getHeightFooter();
     if (isBlurred && !hasComment) {
       return 0;
     }
-    const isSingleComment = getTotalReaction(item) === 1;
-    const commentSectionHeight = getHeightReaction() - getHeightFooter();
-    return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
+    return commentSectionHeight;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `addCommentPreviewHeight` has been simplified to return the comment section height or 0 based on whether the post is blurred and has no comments. However, the condition for a single comment which was present in the old code has been removed. If this condition is still relevant, it should be reintroduced.
```diff
- const addCommentPreviewHeight = () => {
-   const commentSectionHeight = getHeightReaction() - getHeightFooter();
-   if (isBlurred && !hasComment) {
-     return 0;
-   }
-   return commentSectionHeight;
- };
+ const addCommentPreviewHeight = () => {
+   if (isBlurred && !hasComment) {
+     return 0;
+   }
+   const isSingleComment = getTotalReaction(item) === 1;
+   const commentSectionHeight = getHeightReaction() - getHeightFooter();
+   return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->